lib/__tests__/standalone-ext.test.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+const path = require('path');
+const { existsSync } = require('fs');
+const standalone = require('../standalone');
+const safeChdir = require('../testUtils/safeChdir');
+const configColorNamed = require('./fixtures/config-color-named.json');
+
+const fixturesPath = path.join(__dirname, 'fixtures');
+
+describe('extensions', () => {
+	test('array', async () => {
+		const inputFiles = [
+			path.join(fixturesPath, 'no-empty-source.html'),
+			path.join(fixturesPath, 'extensions'),
+		];
+		const { results } = await standalone({
+			files: inputFiles,
+			customSyntax: 'postcss-scss',
+			extensions: ['css', 'scss'],
+			config: {
+				rules: { 'color-named': 'never' },
+			},
+		});
+
+		expect(results).toHaveLength(3);
+		expect(existsSync(path.join(fixturesPath, 'extensions', 'foo.json'))).toBe(true);
```

Comment:
[ask] What does this line `expect(existsSync(...))` expect? Since `foo.json` exists obviously, this expectation seems unneeded.

Reply:
If someone removes `foo.json` (because it's empty) this test must fail.
I could have added `expect(existsSync(path.join(fixturesPath, 'extensions', 'folder-slash', 'file.json'))).toBe(true);` but I considered one is enough.
In short, what we are checking is whether the filter works or not, for that it needs _not_ to pick up existing files.
If someone removes both json files the test would become useless.